packages/lexical-selection/src/lexical-node.ts

```diff
@@ -96,7 +96,7 @@ export function $sliceSelectedTextNodeContent(
   textNode: TextNode,
 ): LexicalNode {
   if (
-    textNode.isSelected() &&
+    textNode.isSelected(selection) &&
```

Comment:
We planning to upstream this fix into the lexical community package?

Reply:
Yes we are

Reply:
https://github.com/facebook/lexical/pull/5380